ios-training-bjung/Common/AppError.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AppError.swift
+//  ios-training-bjung
+//
+//  Created by 鄭 普勤 on 2023/12/12.
+//
+
+import Foundation
+
+public enum AppError: Error, LocalizedError {
```

Comment:
```suggestion
public enum AppError: LocalizedError {
```

`LocalizedError` は `Error` を含んでいる（継承している）ので記載しなくて大丈夫です。

Reply:
@es-kumagai 
ありがとうございます。
localizedDescription を使おうとして、そのまま継承していました。
fbfcf7d1620575e0349f46b1fa66a3b0656b96c4
上記のコミットで対応しました。